server/models/User.ts

```diff
@@ -182,10 +182,9 @@ class User extends Model<InferAttributes<User>, InferCreationAttributes<User>> {
     }
   };
 
-  isRoot = function () {
-    const result = this.hasRole([roles.ADMIN], 1) || this.hasRole([roles.ADMIN], 8686);
-    debug('isRoot?', result);
-    return result;
+  isRoot = function (): boolean {
+    const hasAdminRole = this.hasRole([roles.ADMIN], 1) || this.hasRole([roles.ADMIN], 8686);
```

Comment:
I'm tempted to also move these `1` and `8686` IDs in `config.accountIds.root` to not mess up between dev and prod configs.